src/models/Reasons.ts

```diff
@@ -0,0 +1,7 @@
+import { OtherReason } from './OtherReason';
+
+export interface Reasons {
+
```

Comment:
ew an empty line

Reply:
I just like new lines, makes it more readable for me